plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -45,76 +54,93 @@ export async function start() {
     });
 }
 
-const getSshServiceName = async function (): Promise<string> {
-    /**
-     * Known Che services which can use the SSH key pairs.
-     */
-    const services: CheService[] = [
-        {
-            name: 'vcs',
-            displayName: 'VCS',
-            description: 'SSH keys used by Che VCS plugins'
-        },
-        {
-            name: 'machine',
-            displayName: 'Workspace Containers',
-            description: 'SSH keys injected into all Workspace Containers'
-        }
-    ];
+const getHostName = async function (): Promise<string> {
+    const hostName = await theia.window.showInputBox({ placeHolder: 'Please provide a Host name e.g. github.com' });
+    return hostName ? hostName : '';
+};
 
-    const option: theia.QuickPickOptions = {
-        matchOnDescription: true,
-        matchOnDetail: true,
-        canPickMany: false,
-        placeHolder: 'Select object:'
-    };
-    const sshServiceValue = await theia.window.showQuickPick<theia.QuickPickItem>(services.map(service =>
-        ({
-            label: service.displayName,
-            description: service.description,
-            detail: service.name,
-            name: service.name
-        })), option);
-    if (sshServiceValue) {
-        return Promise.resolve(sshServiceValue.label);
+const updateConfig = function (hostName: string): void {
+    const sshDir = os.homedir() + '/.ssh';
```

Comment:
why not use path.resolve https://nodejs.org/docs/latest/api/path.html#path_path_resolve_paths ? instead of these /

Reply:
fixed